elixir/apps/web/lib/web/live/settings/identity_providers/jumpcloud/components.ex

```diff
@@ -83,7 +83,7 @@ defmodule Web.Settings.IdentityProviders.JumpCloud.Components do
                   </p>
                 </div>
 
-                <div>
+                <div class="hidden">
```

Comment:
We should just remove the div for a hidden field

Reply:
👍